spec/model/user_spec.rb

```diff
@@ -0,0 +1,184 @@
+require "rails_helper"
+include ApplicationHelper
+RSpec.describe User, type: :model do
+  describe "Associations" do
+    it do
+      is_expected.to have_many(:messages).with_foreign_key(:user_send_id).dependent(:destroy)
+      is_expected.to have_many(:active_relationships).with_foreign_key(:follower_id).dependent(:destroy)
+      is_expected.to have_many(:passive_relationships).with_foreign_key(:followed_id).dependent(:destroy)
+    end
+  end
+
+  describe "Validations" do
+    subject {FactoryBot.build(:user)}
+    let(:email) { "user@email.com"}
+    let(:phone) {"0905123123"}
+
+    context "when field name" do
+      it {is_expected.to validate_presence_of(:name)}
+    end
+
+    context "when field email" do
+      it do
```

Comment:
mỗi it một expect thôi e, với thêm mô tả vào

Reply:
em đã fix ạ